From 454d7381979fa61c7b0f11399c481c0b11af586b Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 19 Nov 2008 15:31:31 -0800
Subject: [PATCH 21/21] Staging: comedi: me_daq: fix sparse issues

This fixes the sparse issues that the tool had with the me_daq
driver.

Cc: Michael Hillmann <hillmann@syscongroup.de>
Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/me_daq.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/comedi/drivers/me_daq.c
+++ b/drivers/staging/comedi/drivers/me_daq.c
@@ -234,7 +234,7 @@ static const struct me_board me_boards[]
 		.ao_channel_nbr =	0,
 		.ao_resolution =	0,
 		.ao_resolution_mask =	0,
-		.ao_range_list =	0,
+		.ao_range_list =	NULL,
 		.ai_channel_nbr =	16,
 		/* Analog Input */
 		.ai_resolution =	12,
@@ -257,8 +257,8 @@ COMEDI_PCI_INITCLEANUP(me_driver, me_pci
 /* Private data structure */
 struct me_private_data {
 	struct pci_dev *pci_device;
-	void *plx_regbase;	/* PLX configuration base address */
-	void *me_regbase;	/* Base address of the Meilhaus card */
+	void __iomem *plx_regbase;	/* PLX configuration base address */
+	void __iomem *me_regbase;	/* Base address of the Meilhaus card */
 	unsigned long plx_regbase_size;	/* Size of PLX configuration space */
 	unsigned long me_regbase_size;	/* Size of Meilhaus space */
 
